pyorerun/biorbd_components/model_interface.py

```diff
@@ -44,6 +44,10 @@ def has_meshlines(self) -> bool:
     def mesh_path(self) -> str:
         return self.segment.characteristics().mesh().path().absolutePath().to_string()
 
+    @cached_property
+    def mesh_scale(self) -> str:
```

Comment:
Tell me if i'm wrong:
```python3
def mesh_scale(self) -> np.darray:
```

Reply:
Add a one-line doc to say the format of the output; (3,1) (3,)

Reply:
Yes sorry